api/v1/openapi.yaml

```diff
@@ -2581,6 +2589,11 @@ definitions:
         type: array
         items:
           "$ref": "#/definitions/BackendAddress"
+      backend-weights:
```

Comment:
I assume this needs to be the same length as `backend-addresses`? Can you expand this in the docs?

Reply:
true -- either same length or nothing (which means default weight 1), yeah I can do that

Reply:
removed, now part of `BackendAddress` object